closed/src/java.base/share/native/libjncrypto/NativeCrypto.c

```diff
@@ -99,8 +141,15 @@ typedef int OSSL_RSA_set0_factors_t(RSA *, BIGNUM *, BIGNUM *);
 typedef void OSSL_RSA_free_t (RSA *);
 typedef int OSSL_RSA_public_decrypt_t(int, const unsigned char *, unsigned char *, RSA *, int);
 typedef int OSSL_RSA_private_encrypt_t (int, const unsigned char *, unsigned char *, RSA *, int);
-typedef BIGNUM* OSSL_BN_bin2bn_t (const unsigned char *, int, BIGNUM *);
-typedef void OSSL_BN_set_negative_t (BIGNUM *, int);
+typedef int OSSL_RSA_get0_key_t(const RSA *, const BIGNUM **, const BIGNUM **, const BIGNUM **);;
+typedef BIGNUM* OSSL_BN_bin2bn_t(const unsigned char *, int, BIGNUM *);
+typedef int OSSL_BN_mod_exp_t(BIGNUM *, BIGNUM *, const BIGNUM *, const BIGNUM *, BN_CTX *);
+typedef int OSSL_BN_ucmp_t(BIGNUM *, BIGNUM *);
+typedef int OSSL_BN_zero_t(BIGNUM *a);
+typedef void OSSL_BN_set_negative_t(BIGNUM *, int);
+typedef BN_CTX *OSSL_BN_CTX_new_t();
```

Comment:
Please consistently bind the `*` to the right (to reflect the meaning); write
```
typedef BIGNUM *OSSL_BN_bin2bn_t(const unsigned char *, int, BIGNUM *);
```
instead of
```
typedef BIGNUM* OSSL_BN_bin2bn_t(const unsigned char *, int, BIGNUM *);
```